webapp/src/dogma/features/app-identity/NewAppIdentity.tsx

```diff
@@ -0,0 +1,209 @@
+      }
+      setAppIdentityDetail(response);
+      onNewAppIdentityFormClose();
+      onSecretModalToggle();
+    } catch (error) {
+      dispatch(
+        newNotification('Failed to create a new app identity', ErrorMessageParser.parse(error), 'error'),
+      );
+    }
+  };
+
+  return (
+    <>
+      <Popover placement="bottom" isOpen={isNewAppIdentityFormOpen} onClose={onNewAppIdentityFormClose}>
+        <PopoverTrigger>
+          <Button
+            size="sm"
+            mr={4}
+            rightIcon={<IoMdArrowDropdown />}
+            colorScheme="teal"
+            onClick={onNewAppIdentityFormToggle}
+          >
+            New App Identity
+          </Button>
+        </PopoverTrigger>
+        <PopoverContent minWidth="max-content">
+          <PopoverHeader pt={4} fontWeight="bold" border={0} mb={3}>
+            Create a new app identity
+          </PopoverHeader>
+          <PopoverArrow />
+          <PopoverCloseButton />
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <PopoverBody minWidth="md">
+              <FormControl mb={4}>
+                <FormLabel>Type</FormLabel>
+                <Controller
+                  name="type"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <RadioGroup value={value} onChange={onChange}>
+                      <Stack direction="row" spacing={4}>
+                        <Radio value="TOKEN">Token</Radio>
+                        {mtlsEnabled && <Radio value="CERTIFICATE">Certificate</Radio>}
+                      </Stack>
+                    </RadioGroup>
+                  )}
+                />
+              </FormControl>
+
+              <FormControl isInvalid={errors.appId ? true : false} isRequired>
+                <FormLabel>Application ID</FormLabel>
+                <Input
+                  type="text"
+                  placeholder="my-app-id"
+                  {...register('appId', { pattern: APP_ID_PATTERN })}
+                />
+                <FormHelperText pl={1}>
+                  Register the app identity with a project before accessing it.
+                </FormHelperText>
+                {errors.appId && (
+                  <FormErrorMessage>The first/last character must be alphanumeric</FormErrorMessage>
+                )}
+              </FormControl>
+
+              {selectedType === 'CERTIFICATE' && (
+                <FormControl mt={4} isInvalid={errors.certificateId ? true : false} isRequired>
+                  <FormLabel>Certificate ID</FormLabel>
+                  <Input
+                    type="text"
+                    placeholder="certificate-id"
+                    {...register('certificateId', {
+                      required: selectedType === 'CERTIFICATE',
+                    })}
+                  />
+                  {errors.certificateId && <FormErrorMessage>Certificate ID is required</FormErrorMessage>}
```

Comment:
It would be helpful to provide a message explaining this feature.
```tsx
<FormHelperText pl={1}>
  The ID of the client certificate to be used for mTLS authentication.
</FormHelperText>
```